VBA/Excel-VBA/articles/application-workbookaftersave-event-excel.md

```diff
@@ -14,6 +14,7 @@ ms.date: 06/08/2017
 
 Occurs after the workbook is saved.
 
```

Comment:
Another tip:  GitHub URLs are case sensitive. This means that internal URLs such as "~/**o**ffice-**s**hared-**v**ba/articles/how-autosave-impacts-addins-and-macros" are not valid urls. The actual GitHub folder name is Office-Shared-VBA